src/py42/clients/securitydata.py

```diff
@@ -1,27 +1,30 @@
 import json
 from threading import Lock
 
-from requests.exceptions import HTTPError
-
 from py42.exceptions import Py42ChecksumNotFoundError
 from py42.exceptions import Py42Error
 from py42.exceptions import Py42HTTPError
-from py42.exceptions import Py42ResponseError
+from py42.exceptions import Py42NotFoundError
 from py42.exceptions import Py42SecurityPlanConnectionError
-from py42.exceptions import raise_py42_error
 from py42.sdk.queries.fileevents.file_event_query import FileEventQuery
 from py42.sdk.queries.fileevents.filters.file_filter import MD5
 from py42.sdk.queries.fileevents.filters.file_filter import SHA256
-from py42.settings import debug
 
 
-class SecurityModule(object):
+class SecurityDataClient(object):
     def __init__(
-        self, security_client, storage_client_factory, microservices_client_factory
+        self,
+        security_service,
+        file_event_service,
+        preservation_data_service,
+        saved_search_service,
+        storage_service_factory,
     ):
-        self._security_client = security_client
-        self._storage_client_factory = storage_client_factory
-        self._microservices_client_factory = microservices_client_factory
+        self._security_client = security_service
```

Comment:
I believe this should be `self._security_service = security_service`

Reply:
good catch for clarity, yes